dts/arm/st/stm32f0.dtsi

```diff
@@ -92,6 +92,16 @@
 			status = "disabled";
 			label= "I2C_2";
 		};
+
+		spi1: spi@40013000 {
+			compatible = "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013000 0x400>;
+			interrupts = <25 0>;
```

Comment:
SPI IRQ prio is generally  not 0.
Was 0 put here on purpose and meant to be modified in boards dts?

Reply:
Yes, I set SPI IRQ prio to zero in SOC's dtsi file and meant to override it on application level. Should I change the default value in SOCs level to something else if the SPI IRQ prio is generally not 0? I would like to avoid setting the default SPI IRQ prio in boards level.